docs/content/docs/testing/errors.md

```diff
@@ -0,0 +1,290 @@
+---
+title: Common Fluid Framework Errors
+menuPosition: 5
+---
+
+This article lists the most common errors Fluid developers will encounter. The first steps to resolve the errors are listed. If you get an error that is not included here, please [let us know with a GitHub issue](https://github.com/microsoft/FluidFramework/issues) and we’ll update both the documentation and our guidance. 
+
+### **`<X>` is not a constructor** 
+
+[A generic JavaScript exception](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Not_a_constructor)
+
+**Steps to take** : Investigate the root cause in your application. It’s likely that there is a high-level catch statement around the container operations and the underlying problem is in those operations.  
+
+### **`0x26d`**
```

Comment:
we should not be documenting asserts. asserts should never fire. if they do it is a bug, or the assert should be modified to throw a user readable error instead.

Reply:
If asserts do fire and customers are asking about them, then we need to document them. I agree with what you're saying, but this document needs to apply to older releases as well, where even if we change an assert to be a user readable error, not everyone would benefit from that change.

Reply:
i'd rather just say this document applies to 2.0 forward, and not worry about the past, that would resolve many of the issue below.

Reply:
@pk-pranshu whether we can say this applies to 2.0 forward is probably a pm decision, but i'm not sure there is another option. this document probably needs versioning and AFIK we don't have one for 1.0

Reply:
@anthony-murphy - We have customers who asked about the asserts and what they mean multiple times so we decided to add that. I do like the idea of versioning as most of our customers are on 1.0 at this time. Eventually once we deprecate 1.0 and 2.0 is GA, we can just focus on those. Let me think more about how to best represent this based on versioning. I think that should apply to all of our docs and should not be exclusive to errors guide.

Reply:
Outside of versioning, i think we should do the following for all asserts before documentation such that we use this process as a way to both improve the product and the experience for our customers:

1. Patch the code such that we no longer fire an assert, but instead provide user facing error
2. Document the user facing error, with a small addendum that older versions could see the assert code